app/controllers/find/cookie_preferences_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class Find::CookiePreferencesController < ApplicationController
+  layout "find_layout"
+  skip_before_action :authenticate
```

Comment:
Do we need this? There's no auth for Find

Reply:
Yep, I have switched to inheriting from Find:ApplicationController